clippy_lints/src/methods/mod.rs

```diff
@@ -3073,7 +3073,7 @@ fn lint_search_is_some<'tcx>(
             then {
                 let msg = "called `is_some()` after calling `find()` \
                     on a string. This is more succinctly expressed by calling \
-                    `contains()`";
+                    `contains()`.";
```

Comment:
@flip1995 I added back in the period after `contains()` to match the other cases of `search_is_some` (they all end with a period).  Let me know if you do not want that and I will remove it and re-run `update-all-references`.

By the way, thank you for you patience with me!  Sorry if my questions are all basic things.

Reply:
Oh they're also in other places? Can you remove them there too, please? Rust error/hint/suggestion messages are always without punctuation at the end and start lower-case.

Reply:
You could also remove the second part of this message here completely and replace the `"try this"` below with ``use `contains()` instead``.

Reply:
@flip1995 Ok, I think they should be correctly updated.  I also cleaned up the other lints in `search_is_some` that suggested using `any()`.